src/activities/Elsa.Activities.AzureServiceBus/FodyWeavers.xsd

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If these files are generated, can't we exclude the from git? Assuming they'll be generated upon build.

Reply:
I didn't assume that these would be constantly recreated.

Reply:
Do you know at which point they are created? Looking at this [.gitignore template](https://raw.githubusercontent.com/github/gitignore/master/VisualStudio.gitignore), I take it is fine to exclude them.

Reply:
I also think it's better. I didn't think of it at first. 

The .gitignore contains more entries than the .gitignore from this repository. Maybe you should take the entries to solve future problems.

If you could please take a look at my other pull request, then I could recreate the fork and make the change this week.

Reply:
That's fine, the visual studio ignore is there as a convenience, but as you see it also contains entries of things that are very unlikely to be ever used in this project, such as xamarin and Visual C++. It's fine to add new entries as needed.

I'll check out your other PR!